lib/Runtime/ByteCode/ByteCodeCacheReleaseFileVersion.h

```diff
@@ -4,6 +4,6 @@
 //-------------------------------------------------------------------------------------------------------
 // NOTE: If there is a merge conflict the correct fix is to make a new GUID.
 
-// {9B3A48D4-C8EF-46CF-A473-14CB0DA3244F}
+// {3EB30F58-66E0-404B-BAE8-1D3CD23F866F}
 const GUID byteCodeCacheReleaseFileVersion =
-{ 0x9B3A48D4, 0xC8EF, 0x46CF, { 0xA4, 0x73, 0x14, 0xCB, 0x0D, 0xA3, 0x24, 0x4F } };
+{ 0x3EB30F58, 0x66E0, 0x404B, { 0xBA, 0xE8, 0x1D, 0x3C, 0xD2, 0x3F, 0x86, 0x6F } };
```

Comment:
just wanted to clarify that there is no need to generate the intl bytecode cache?

Reply:
@MSLaguana recommended to regenerate it in a comment further up.